package.json

```diff
@@ -28,7 +28,7 @@
     "start:matrixsender": "node --require source-map-support/register lib/App/MatrixSenderApp.js",
     "test": "mocha -r ts-node/register tests/init.ts tests/*.ts  tests/**/*.ts",
     "lint": "yarn run lint:js && yarn run lint:rs",
-    "lint:js": "eslint -c .eslintrc.js src/**/*.ts",
+    "lint:js": "eslint -c .eslintrc.js 'src/**/*.ts' 'tests/**/*.ts' 'web/**/*.ts' 'web/**/*.tsx'",
```

Comment:
For `**` to work reliably the string has to be quoted. Otherwise, it might get resolved by the operating system.